src/provider/qti.js

```diff
@@ -251,6 +251,7 @@ var qtiProvider = {
             const currentItem = self.getCurrentItem();
             const options = self.getOptions();
             const skipPausedAssessmentDialog = !!options.skipPausedAssessmentDialog;
+            const { partiallyAnsweredIsAnswered } = options.review;
```

Comment:
Is it possible for `options.review` to be undefined? If yes, should check for it.

Reply:
Ok, you are right. It's checked in some places like https://github.com/oat-sa/tao-test-runner-qti-fe/blob/develop/src/plugins/navigation/review/review.js#L164 or `src/helpers/stats.js` in this PR, but it shouldn't happen.